src/asciidoc/docs/getting-started/developer-updates/2021MarUpdates.adoc

```diff
@@ -0,0 +1,74 @@
+= March 2021 updates
+:page-slug: 2021-mar-updates
+:page-description: Brief digest of updates in March 2021 to the Koji platform and developer site.
+
+This digest summarizes updates to the Koji platform, packages, and developer site in March 2021.
+
+include::../../_includes/tip-discord-server.adoc[tags=*]
+
+// tag::highlights[]
+== March 17
+
+* The following usability improvements are now available on the Koji platform.
+** Creators can now automatically upgrade their Kojis to the latest version of your code.
+When an upgrade is available, the creater can tap *Upgrade this Koji* on the Koji's analytics page.
+** Your Koji profile page has been updated to a new link-in-bio profile that is powered by a Koji template.
+The new profile builder features fully customizable backgrounds, buttons, and fonts and provides the same powerful analytics that are built into your Kojis.
++
+image::profileMar2021.png[]
+
+** The navigation experience in Kojis has been improved with updates to the modal windows for embedded Kojis, removal of the confirmation prompt when linking to another website, and support for `koji.to` links.
+** The onboarding flow for new users has been redesigned for a better welcome experience to Koji.
+// end::highlights[]
+** The experiences for publishing a remix, using asset packs and associated editing tools, and selecting a color have been redesigned for better usability.
+System alerts and the loading spinner have also been updated.
+** The Koji theme is no longer available as an account setting.
+All accounts now default to the dark theme preference.
```

Comment:
This only happens if your device is set to dark mode. Normal users will see the light theme.

Reply:
@sthielen Thanks, updated! Let me know if any more changes are required.